account_sdk/src/vars.rs

```diff
@@ -0,0 +1,32 @@
+#[cfg(target_arch = "wasm32")]
+use {js_sys, web_sys};
+
+pub fn get_cartridge_api_url() -> String {
```

Comment:
Would prefer to pass this as an arg, like we do for `rpc_url`, rather than get it from the env